style/style.css

```diff
@@ -0,0 +1,154 @@
+/* Typography */
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#container {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #dbdada;
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  padding-bottom: 30px;
+}
+
+#title {
+  font-size: 45px;
+  margin-top: 50px;
+  font-weight: 700;
+  text-align: center;
+  color: #8a2be2;
+}
+
+cite {
+  color: black;
+}
+
+#image {
+  width: 80%;
+  margin-top: 40px;
+  margin-bottom: 25px;
+  max-width: 900px;
+}
+
+#img-caption {
+  font-size: 17px;
+  margin-bottom: 30px;
+  text-align: center;
+}
```

Comment:
Please avoid using ID selectors on multiple selectors. It is not the [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md). When your codebase grows, it can become very messy.